docker/.env.example

```diff
@@ -596,6 +596,10 @@ SSRF_PROXY_HTTP_URL=http://ssrf_proxy:3128
 # SSRF Proxy server HTTPS URL
 SSRF_PROXY_HTTPS_URL=http://ssrf_proxy:3128
 
+# Enable or disable the X-Forwarded-For Proxy Fix middleware from Werkzeug to respect X-* headers to redirect clients
+# Reference: https://werkzeug.palletsprojects.com/en/3.0.x/middleware/proxy_fix/
```

Comment:
Remove the comment here, as it's simply repeating the config definition in DifyConfig.